DigiMeSDK/Core/Classes/Network/DMEAPIClient.m

```diff
@@ -242,7 +242,7 @@ - (DMERequestFactory *)requestFactory
 {
     if (!_requestFactory)
     {
-        _requestFactory = [[DMERequestFactory alloc] initWithConfiguration:self.config];
+        _requestFactory = [[DMERequestFactory alloc] initWithConfiguration:_config];
```

Comment:
Why are we directly accessing `_config` here instead of `self.config`? We might miss out on config's initial setting as it is lazily loaded.